DXMainClient/DXGUI/Generic/ExtrasWindow.cs

```diff
@@ -64,13 +65,17 @@ private void BtnExStatistics_LeftClick(object sender, EventArgs e)
 
         private void BtnExMapEditor_LeftClick(object sender, EventArgs e)
         {
-            Process.Start(ProgramConstants.GamePath + ClientConfiguration.Instance.MapEditorExePath);
+            Process.Start(SafePath.CombineFilePath(ProgramConstants.GamePath, ClientConfiguration.Instance.MapEditorExePath));
             Enabled = false;
         }
 
         private void BtnExCredits_LeftClick(object sender, EventArgs e)
         {
-            Process.Start(MainClientConstants.CREDITS_URL);
+            using var _ = Process.Start(new ProcessStartInfo
+            {
+                FileName = MainClientConstants.CREDITS_URL,
+                UseShellExecute = true
+            });
```

Comment:
Not sure why the two are different?

Reply:
One is launching an executable while the other is launching a URL. UseShellExecute = true is only needed when not directly launching an executable.